console/bridge.go

```diff
@@ -56,7 +56,7 @@ func (b *bridge) NewAccount(call otto.FunctionCall) (response otto.Value) {
 		err      error
 	)
 	switch {
-	// No password was specified, prompt the user for it
+	// No password was specified, prompt the user for it.
```

Comment:
Please only add periods to method documentations. Internal comments are just notes without periods at the end (unless they are many senteces). That's the general style of the codebase currently.

Reply:
Ok, I got that. I've updated the codes  per your comments